crates/stages/src/pipeline/set.rs

```diff
@@ -1,6 +1,6 @@
-use crate::Stage;
 use reth_db::database::Database;
 use reth_primitives::stage::StageId;
+use reth_stages_api::Stage;
```

Comment:
this should be moved to the api crate

Reply:
How about "test_utils"? Is it good idea to move them into "stages_api"?

Reply:
imo test-utils should stay in reth-stages

Reply:
It makes sense. But the tests of `pipeline` still relay on `reth_stages::util_utils::TestStage`, and while `pipeline` has been moved into `reth_stages_api`, I moved `reth_stages::util_utils::TestStage` into `reth_stages_api::util_utils::TestStage`, is there any problem?